extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeMusicSearchExtractor.java

```diff
@@ -133,7 +133,7 @@ public String getUrl() throws ParsingException {
     public String getSearchSuggestion() throws ParsingException {
         final JsonObject itemSectionRenderer = initialData.getObject("contents").getObject("sectionListRenderer")
                 .getArray("contents").getObject(0).getObject("itemSectionRenderer");
-        if (itemSectionRenderer == null) {
+        if (itemSectionRenderer.equals(JsonUtils.EMPTY_OBJECT)) {
```

Comment:
That works, however you could just do `.isEmpty()` instead, which would be slightly faster, but mainly is more readable imho